### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/path-params-numeric-validations.md`

### PR DESCRIPTION
**Pull request by lbmendes**
Hello everyone!

This PR adds the Portuguese translation of tutorial/path-params-numeric-validations

### CODE REVIEW

File: docs/pt/docs/tutorial/path-params-numeric-validations.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+!!! nota
+    Um parâmetro de rota é sempre obrigatório como se fosse parte da rota.
+    
+    Logo, você deve declará-lo com `...` para marcá-lo como obrigatório.
+
+    Mesmo que você declare-o como `None` ou defina um valor padrão, isso não teria efeito algum, o parâmetro ainda seria obrigatório.
+
+## Ordene os parâmetros de a acordo com sua necessidade
+
+Suponha que você quer declarar o parâmetro de consulta `q` como uma `str` obrigatória.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/path-params-numeric-validations.md:32]:
```suggestion
Suponha que você declare o parâmetro de consulta `q` como uma `str` obrigatória.
```

File: docs/pt/docs/tutorial/path-params-numeric-validations.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+E você não precisa declarar mais nada em relação a este parâmetro, então você não precisa necessariamente usar `Query`.
+
+Mas você ainda precisa usar `Path` para o parâmetro de rota `item_id`.
+
+O Python vai acusar se você colocar um elemento com um valor padrão definido antes de outro que não tenha um valor padrão.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/path-params-numeric-validations.md:38]:
```suggestion
O Python irá acusar se você colocar um elemento com um valor padrão definido antes de outro que não tenha um valor padrão.
```

File: docs/pt/docs/tutorial/path-params-numeric-validations.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    `Query`, `Path`, e outras que você verá a frente são subclasses de uma classe `Param` comum (que você não precisa usar).
+
+    E todas elas compartilham todos esses mesmos parâmetros de validação adicional e metadados que você viu.
+
+!!! note "Detalhes Técnicos"
+    Quando você importa `Query`, `Path` e outras de `fastapi`, elas são na verdade funções.
+
+    Que quando chamadas, retornam instâncias de classes de mesmo nome.
+
+    Então, você importa `Query`,que é uma função. E quando você a chama, ela retorna uma instância de uma classe também chamada `Query`.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/path-params-numeric-validations.md:118]:
```suggestion
    Então, você importa `Query`, que é uma função. E quando você a chama, ela retorna uma instância de uma classe também chamada `Query`.
```

**Review by LorhanSohaky — CHANGES_REQUESTED** [review]
I think with these suggestions the text is more natural

File: docs/pt/docs/tutorial/path-params-numeric-validations.md (hunk 3 above)
Comment (lbmendes) [reply docs/pt/docs/tutorial/path-params-numeric-validations.md:118]: Thanks for the suggestions @LorhanSohaky , little details that make the text better to read. I will put the suggestions in a new commit.

File: docs/pt/docs/tutorial/path-params-numeric-validations.md (hunk 1 above)
Comment (lbmendes) [reply docs/pt/docs/tutorial/path-params-numeric-validations.md:32]: In this case specifically I think your suggestion has a slightly change in the original idea of a person thinking in declare the query param.

Comment (LorhanSohaky) [reply docs/pt/docs/tutorial/path-params-numeric-validations.md:32]: what do you think of "Suponha que você queira declarar o parâmetro de consulta `q` como uma `str` obrigatória"?

Comment (lbmendes) [reply docs/pt/docs/tutorial/path-params-numeric-validations.md]: That's nice! I created a new commit with this suggestion, thank you!

**Review by LorhanSohaky — APPROVED** [review]

File: docs/pt/docs/tutorial/path-params-numeric-validations.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+```Python hl_lines="8"
+{!../../../docs_src/path_params_numeric_validations/tutorial003.py!}
+```
+
+## Validações numéricas: maior que ou igual
+
+Com `Query` e `Path` (e outras que você verá mais tarde) você pode declarar restrições de string, e também restrições numéricas.
+
+Aqui, com `ge=1`, `item_id` precisará ser um numéro inteiro maior que ("`g`reater than") ou igual ("`e`qual") a 1.
```
Comment (peidrao) [inline docs/pt/docs/tutorial/path-params-numeric-validations.md:66]:
```suggestion
Aqui, com `ge=1`, `item_id` precisará ser um número inteiro maior que ("`g`reater than") ou igual ("`e`qual") a 1.
```

**Review by peidrao — CHANGES_REQUESTED** [review]

File: docs/pt/docs/tutorial/path-params-numeric-validations.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+Você pode declarar todos os parâmetros da mesma maneira que na `Query`.
+
+Por exemplo para declarar um valor de metadado `title` para o parâmetro de rota `item_id` você pode digitar:
+
+```Python hl_lines="10"
+{!../../../docs_src/path_params_numeric_validations/tutorial001.py!}
+```
+
+!!! nota
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/path-params-numeric-validations.md:23]:
```suggestion
!!! note "Nota"
```

File: docs/pt/docs/tutorial/path-params-numeric-validations.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    Um parâmetro de rota é sempre obrigatório, como se fizesse parte da rota.
+    
+    Então, você deve declará-lo com `...` para marcá-lo como obrigatório.
+
+    Mesmo que você declare-o como `None` ou defina um valor padrão, isso não teria efeito algum, o parâmetro ainda seria obrigatório.
+
+## Ordene os parâmetros de a acordo com sua necessidade
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/path-params-numeric-validations.md:30]:
```suggestion
## Ordene os parâmetros de acordo com sua necessidade
```

**Review by stlucasgarcia — CHANGES_REQUESTED** [review]

File: docs/pt/docs/tutorial/path-params-numeric-validations.md (hunk 6 above)
Comment (lbmendes) [reply docs/pt/docs/tutorial/path-params-numeric-validations.md]: Thanks for the review and suggestions @lsglucas!

**Review by stlucasgarcia — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thanks @lbmendes! :rocket: 

And thanks for the reviews @peidrao, @LorhanSohaky, @lsglucas :bow: :bow:

**Pull request merged**